addons/mrp/models/mrp_workorder.py

```diff
@@ -113,7 +113,6 @@ class MrpWorkorder(models.Model):
     final_lot_id = fields.Many2one(
         'stock.production.lot', 'Lot/Serial Number', domain="[('product_id', '=', product_id)]",
         states={'done': [('readonly', True)], 'cancel': [('readonly', True)]})
-    tracking = fields.Selection(related='production_id.product_id.tracking')
```

Comment:
why ?

Reply:
This is the duplicate field so I have removed.. 
https://github.com/odoo/odoo/blob/master/addons/mrp/models/mrp_workorder.py#L49